### DISCUSSION TITLE
Change Budibase Installation using other database (not using CouchDB)

### DISCUSSION THREAD

**Discussion by fsugiharja** [Budibase/budibase#12286] [Help] [answered]
Hi all,

Is it possible to change the installation of budibase not using couch db?

Thanks

**Comment by mike12345567** [comment, marked as answer]
Hi @fsugiharja - CouchDB is fundamental to how Budibase functions - it is required for the system to work.

**Comment by fsugiharja** [comment]
> Hi @fsugiharja - CouchDB is fundamental to how Budibase functions - it is required for the system to work.

so, couchdb is a mandatory to run budibase on self-host?
can't change to use other no-sql database (such as mongo)?

**Reply by ConorWebb96** [reply]
Yes that is correct, thats exactly what Mike means.
